### PR TITLE
[Doc Improvement][Copilot ME Plugin for M365 private preview]

### PR DESCRIPTION
**Pull request by v-shalinir**
Added note for private preview

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/high-quality-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2a4d86e90bf593df979ed13abc3ad280add8bcc8/msteams-platform%2Fmessaging-extensions%2Fhigh-quality-message-extension.md) | 92 | 43 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b7c3ee7d-5755-44ea-bf8e-afae1a01e387) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/high-quality-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/88e035a18623efbbaa694e9dd152d983062c7683/msteams-platform%2Fmessaging-extensions%2Fhigh-quality-message-extension.md) | 92 | 42 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/927d0df0-6cab-4250-810e-cbf909729564) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/messaging-extensions/high-quality-message-extension.md [hunk 1]
```diff
@@ -767,24 +767,27 @@ For a plugin to be validated, invoked, and work seamlessly, ensure that it meets
 
 ## Message extensions Plugins in Copilot for Microsoft 365
 
+> [!IMPORTANT]
+> Message extensions Plugins in Copilot for Microsoft 365 are in limited private preview. More details to be published after a public preview is announced.
+
 Copilot extensions customize and extend the Copilot for Microsoft 365 experience by bringing more skills and knowledge to Copilot for a personalized user experience. By using plugins, which are a subset of Copilot extensions, users can integrate additional capabilities into Copilot by interacting with third-party applications, whether for retrieving or modifying information within those apps. For instance, message extension plugins facilitate searching for data in other applications so that Copilot can present it upon request when the plugin is activated.
```
Comment (isvargasmsft) [inline msteams-platform/messaging-extensions/high-quality-message-extension.md:773]: Two observations:
1. This section is about Copilot in M365 apps, specifically in Word and PowerPoint. Saying M365 Copilot is misleading, as MEs are GAd and available to customers as plugins in M365 Copilot today. But in private preview in Copilot in Word and PowerPoint. Can we add this clarification to the title of the section and also to the note, please? Suggestions:
For the title: Message extensions plugins in Copilot for M365 applications
For the note: Message extensions plugins in Copilot for M365 applications are in limited private preview for Word and PowerPoint. More details to be published after a public preview is announced.
2. In the title we are capitalizing the word "Plugins", can we use small caps there, please?

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/high-quality-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/75969e8fb0bfa8c2e8744d72533714112c892f7b/msteams-platform%2Fmessaging-extensions%2Fhigh-quality-message-extension.md) | 92 | 43 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4b8cfd81-97cb-4edb-a2fd-000385be5f66) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/messaging-extensions/high-quality-message-extension.md (hunk 1 above)
Comment (v-shalinir) [reply msteams-platform/messaging-extensions/high-quality-message-extension.md]: @isvargasmsft Comments are incorporated in the PR.

**Review by isvargasmsft — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/high-quality-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b1cbc6c53cccefefc97d9078350a6f244c8ad249/msteams-platform%2Fmessaging-extensions%2Fhigh-quality-message-extension.md) | 92 | 42 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c60a6996-e9cb-494f-b220-fc7680337c45) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**